gax-java/gax/src/main/java/com/google/api/gax/rpc/Watchdog.java

```diff
@@ -71,19 +75,20 @@ public final class Watchdog implements Runnable, BackgroundResource {
   private final ConcurrentHashMap<WatchdogStream, Object> openStreams = new ConcurrentHashMap<>();
 
   private final ApiClock clock;
-  private final Duration scheduleInterval;
+  private final java.time.Duration scheduleInterval;
   private final ScheduledExecutorService executor;
   private ScheduledFuture<?> future;
 
   /** returns a Watchdog which is scheduled at the provided interval. */
   public static Watchdog create(
-      ApiClock clock, Duration scheduleInterval, ScheduledExecutorService executor) {
+      ApiClock clock, java.time.Duration scheduleInterval, ScheduledExecutorService executor) {
```

Comment:
This is a public static method

Reply:
I missed this. I added an overload to allow both types.

Reply:
Done https://github.com/googleapis/sdk-platform-java/pull/1872#discussion_r1628618489